src/posts/siparcs_2022/marshall_siparcs_2022.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Reflection on 2022 NCAR summer internship: "Leveraging Xarray for reproducible and scalable analysis of remote sensing data in the cloud"'
```

Comment:
Could the title be shortened?


<img width="1104" alt="Screenshot 2022-10-28 at 11 07 24 AM" src="https://user-images.githubusercontent.com/13301940/198693376-9e3f71ba-e6a3-464f-adec-33e63ac7a0a0.png">

Reply:
```suggestion
title: 'Reflections on my NCAR summer internship working with Xarray and satellite data"'
```

Hmm.. not the biggest fan of this suggestion

Reply:
thanks @andersy005 and @dcherian ! happy to make more changes if needed as well